frontend/packages/console-plugin-sdk/src/withExtensions.tsx

```diff
@@ -92,7 +92,7 @@ export const withExtensions = <
 };
 
 type ExtensionProps<E extends Extension> = {
-  [propName: string]: E[];
+  [propName: string]: E[] | LoadedExtension<E>[];
```

Comment:
FYI - this type union is to allow `withExtension` users to keep injecting extension types instead of `LoadedExtension` types.